### DISCUSSION TITLE
subscriber status is incorrect when MME Pooling is enabled

### DISCUSSION THREAD

**Discussion by vinothinfinity** [magma/magma#15232] [General]
We have enabled MME Pool in a network with 2 AGWs

We see two issues in HA subscriber status

1. When UE moved from AGW1 to AGW2. UE is idle in AGW1, connected in AGW2. NMS is just shuffling between idle and connected. I assume this is because status from AGWs are getting overwritten alternatively in Orc8r

2. Consider same above case, UE moved from AGW1 to AGW2 and now UE got deleted in AGW1. We just end up deleting directory record in subscriber record in orc8r. AGW2 doesn't send same status again unless there is a change so in NMS we see it as unregistered i.e no location history

This is tested in v1.6.1. Let me know if any discussion on same topic or if some fix went latest master for this. Thanks

**Comment by panyogesh** [comment]
It was tested in v1.6.1 but as its quite hold we can get some relevant information in release notes. Will appreciate if you can test on latest release line >= 1.8 and confirm if issue still exists.

**Reply by vinothinfinity** [reply]
sure. Thank You.
